pennylane_lightning_kokkos/_serialize.py

```diff
@@ -277,7 +277,7 @@ def _serialize_ops(
     sv_py = LightningKokkos_C64 if use_csingle else LightningKokkos_C128
 
     for o in tape.operations:
-        if isinstance(o, (BasisState, QubitStateVector)):
+        if isinstance(o, (BasisState, StatePrep)):
```

Comment:
Just a note that `QubitStateVector` should still work, so it isn't a deprecation/replacement, more that we'll just mainly show `StatePrep` from now on.

Reply:
Thanks @trbromley , I didn't get that. I'll make the necessary changes.

Reply:
Since its an `isinstance` check, `StatePrep` is fine, because `QubitStateVector` inherits from `StatePrep`.